lib/ansible/modules/cloud/amazon/route53.py

```diff
@@ -413,7 +413,7 @@ def main():
         alias=dict(required=False, type='bool'),
         alias_hosted_zone_id=dict(required=False),
         alias_evaluate_target_health=dict(required=False, type='bool', default=False),
-        value=dict(required=False, type='list', default=[]),
+        value=dict(required=False, type='list'),
```

Comment:
If state is `get` and value isn't provided I think [line 518 ](https://github.com/ansible/ansible/pull/32297/files#diff-ee65e55980e15661bf789266142f496cR518) will fail now (because value_in will be None rather than an empty list). Perhaps [line 461 ](https://github.com/ansible/ansible/pull/32297/files#diff-ee65e55980e15661bf789266142f496cL461) could be `value_in = module.params.get('value') or []`?

Reply:
ahh, you are right. Sorry about that. Thanks for reviewing.